archaius2-archaius1-bridge/src/main/java/com/netflix/archaius/bridge/StaticAbstractConfiguration.java

```diff
@@ -35,21 +59,37 @@ public static void reset() {
 
     @Override
     public boolean isEmpty() {
+        if (delegate == null) {
+            System.err.println("[isEmpty()] StaticAbstractConfiguration not initialized yet.");
```

Comment:
If the config is broken would it make more sense to throw an exception? The stderr maybe directed somewhere other than the primary log and so it seems like this would be easy to miss. And if the application isn't configured correctly failing outright would probably be better than coming up in an odd state with strange or broken behavior because configuration wasn't working correctly.